api/v1/standalone-dns-proxy/standalone-dns-proxy.proto

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+syntax = "proto3";
+
+package standalonednsproxy;
+
+option go_package = "github.com/cilium/cilium/api/v1/standalonednsproxy";
+
+service FQDNData {
+  rpc SubscribeToDNSPolicies(stream  DNSPoliciesResult) returns (stream DNSPolicies) {}
+  rpc UpdatesMappings(FQDNMapping) returns (UpdatesMappingsResult) {}
+}
+
+message DNSPoliciesResult {
+    bool success = 1;
+    string request_id = 2;
+}
+
+message FQDNMapping {
+    string FQDN = 1;
```

Comment:
nit: let's try to be consistent in our use of casing - FQDN does not need to be capitalized here (neither does IPs or TTL)

Do we guarantee this to be a FQDN in the sense of whatever DNS RFC defined it?